docs/theory/force_calibration/force_calibration.rst

```diff
@@ -0,0 +1,357 @@
+
+There are two ways to perform such averaging:
+
+- The first is to split the time series into windows of equal length, compute the power spectrum for each chunk of data and averaging these. This procedure is referred to as windowing.
+- The second is to calculate the spectrum for the full dataset and downsampling the obtained spectrum in the spectral domain by averaging adjacent bins according to :cite:`berg2004power`. This procedure is referred to as blocking.
+
+We use the blocking method for spectral averaging.
+This allows us to reject noise peaks at high resolution prior to blocking.
+Note however, that the error incurred by this blocking procedure depends on :math:`n_b`, the number of points per block, :math:`\Delta f`, the spectral resolution and inversely on the corner frequency :cite:`berg2004power`.
+
+Setting the number of points per block too low would result in a bias from insufficient averaging :cite:`berg2004power`.
+In practice, one should use a high number of points per block (:math:`n_b >> 100`), unless a very low corner frequency precludes this.
+In such cases, it is preferable to increase the measurement time.
+
+Passive calibration
+-------------------
+
+Passive calibration is also often referred to as thermal calibration and involves calibration without moving the trap or stage.
+In passive calibration, the Brownian motion of the bead in the trap is analyzed in order to find calibration factors for both the positional detection as well as the force.
+
+To fit passive calibration data, we will use a model based on a number of publications by the Flyvbjerg group :cite:`berg2004power,tolic2004matlab,hansen2006tweezercalib,berg2006power`.
+This model can be found in :class:`~.PassiveCalibrationModel`.
+The most basic form of passive calibration starts by fitting the following equation to the power spectrum:
+
+.. math::
+
+    P(f) = \frac{D_\mathrm{measured}}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)} g(f, f_\mathrm{diode}, \alpha) \quad \mathrm{[V^2/Hz]}
+
+where :math:`D_\mathrm{measured}` corresponds to the diffusion constant (in `volts`), :math:`f` the frequency and :math:`f_c` the corner frequency.
+The first part of the equation is the same as before and represents the part of the spectrum that originates from the physical motion of the bead.
+The second term :math:`g` takes into account the slower response of the position detection system and is given by:
+
+.. math::
+
+    g(f, f_\mathrm{diode}, \alpha) = \alpha^2 + \frac{1 - \alpha ^ 2}{1 + (f / f_\mathrm{diode})^2}
+
+Here :math:`\alpha` corresponds to the fraction of the signal response that is instantaneous, while :math:`f_\mathrm{diode}` characterizes the frequency response of the diode.
+Note that not all sensor types require this second term.
+
+To convert the parameters obtained from this spectral fit to a trap stiffness, the following is computed:
+
+.. math::
+
+    \kappa = 2 \pi \gamma_0 f_c \quad \mathrm{[N/m]}
```

Comment:
Personally, I like 'em by the side, since it makes what goes with what very clear imho. We could consider increasing the spacing; but it might start looking weird at some point as well.